sklearn/svm/_classes.py

```diff
@@ -1332,7 +1332,7 @@ class NuSVR(RegressorMixin, BaseLibSVM):
         default 0.5 will be taken.
 
     C : float, default=1.0
-        Penalty parameter C of the error term. Must be non-negative.
```

Comment:
Should this have been removed?

Reply:
Uhm it does not make sense indeed. I don't understand why we expose the `C` parameter here.

I think this is better to let it as-is in this PR and to make a separate PR to remove (or deprecate) the parameter C from `NuSVR`. It will require adding an entry in the changelog. Deprecation will be better than just removing the parameter. It will be less brutal for the end-user.

Reply:
Ah right. I was only reading the nu-SVC algorithm this morning. So I got confused. Thanks for the explanation.

Reply:
My comment was that the `Must be non-negative.` part that I added was removed in @jeremiedbb's commit. I wanted to confirm that it was intentional, since the parameter validation now requires non-negative values for `C`.

Reply:
Yes, it was just a nitpick, to keep parameter description consistent accross svm estimators. We'll need to make a pass on all docstrings at some point anyway to properly write the accepted range of values.

Reply:
Oh, during the sprint at EuroPython I believe @glemaitre mentioned to update the docstrings to match the parameter validation. Good to know that I shouldn't update them on future PRs for this meta-issue. It would be a good idea to mention this in https://github.com/scikit-learn/scikit-learn/issues/23462 to be clear for others though.

Reply:
No no, it's fine to update the docstrings. It's just that here you only did the C description for one svm estimators so the others svm estimators would have a different description which I didn't find very user friendly.

Reply:
Gotcha.